Program.cs

```diff
@@ -153,10 +161,56 @@ static void ExecuteCommand(Dictionary<string, Account> accounts)
             }
         }
 
+        static void ImportFile(Dictionary<String, Account> accounts, string input)
+        {
+            string operand = input.Substring(12);
+            logger.Info($"User requested file import for {operand}.");
+
+            try
+            {
+                string extension = input.Substring(input.LastIndexOf("."));
+                switch (extension)
+                {
+                    case ".csv":
+                        accounts = ReadCSV(operand);
```

Comment:
I'm assuming this will just overwrite the accounts? What if you want to add transactions from a new file to the accounts you already have? 🙂